charts/helmet/README.md

```diff
@@ -330,12 +330,33 @@ In case you want to add extra environment variables (useful for advanced operati
 
 ```yaml
 envVars:
-  - name: LOG_LEVEL
-    value: error
+  LOG_LEVEL: error
 ```
 
 Alternatively, you can use a ConfigMap or a Secret with the environment variables. To do so, use the `.envVarsConfigMap` or the `envVarsSecret` properties.
 
+#### Migrating from previous version
```

Comment:
Can you please mention exact versions? Migrating from `0.x` to `1.x` for example.